src/test/java/com/thoughtworks/pacman/ui/screens/LostScreenTest.java

```diff
@@ -17,30 +17,30 @@
     @Mock
     private Graphics2D graphics;
 
-    @Test
+    /*@Test
```

Comment:
Neden bu testi uncomment ediyorsun?

Gereksizse silmek daha mantıklı değil mi?

Ayrıca yeni yazdığın kodlarla ilgili eklediğin yeni testler var mı? Sanki sadece testler kırılmasın diye düzeltmeler yapmıssın ama yeni test yazmamışsın.